themes/tiddlywiki/vanilla/base.tid

```diff
@@ -3524,3 +3520,13 @@ span.tc-translink > a:first-child {
 .tc-tiny-v-gap-bottom {
 	margin-bottom: 3px;
 }
+
+/* Float clearfix. Needs to be used by elements that allow float:right in their content.
+** So the floating element does not overflow
+*/
+
+.clearfix:after {
```

Comment:
Why not use `tc-clearfix`?

Reply:
hmm, I thought if users search for `clearfix` at "search-engines" they get a lot of information because everyone uses that term. -- I did test it and it actually ignores the `tc-` part and shows info about clearfix .. So it would be OK to change it. 

Should I change it?

Reply:
Yes I think consistency is important here